stage #2/module #1 Modern Java/lesson #2. Structuring and encapsulation/content.md

```diff
@@ -121,9 +121,24 @@ Module types:
 - Automated Modules - added to the module path jars
 - Unnamed Modules - added to the classpath jars and classes
 
+## Hidden classes
+
+Hidden classes is a feature of JDK 15 that’s most interesting to framework developers.
+
+It allows them to create classes that cannot be used directly by the bytecode of other classes, they are intended for use by the framework itself that generate classes at run time and use them indirectly, via reflection.
```

Comment:
I am not sure, may be 'runtime' is better than 'run time'

Reply:
fixed